project_testing/testing_baseline.yml

```diff
@@ -42,6 +43,8 @@ workflow_generator:
       include_timeseries_end_use_consumptions: true
       include_timeseries_emissions: true
       include_timeseries_total_loads: false
+      output_variables:
+        - name: Zone Mean Air Temperature
```

Comment:
That's a weird choice since you can get these temperatures using the built-in `include_timeseries_temperatures`.

Reply:
I previously had it set to "Zone People Occupant Count" but it was showing up as all zeros for some simulations. Any suggestions? (Also, with consideration to file size.)

Reply:
That's weird. Occupants.tsv suggests that every home should have at least one occupant. Can you investigate why that is?